content/introduction/concepts.md

```diff
@@ -0,0 +1,128 @@
+1. [Spacelift](https://spacelift.io/): By using the [terraform-spacelift-cloud-infrastructure-automation module](https://github.com/cloudposse/terraform-spacelift-cloud-infrastructure-automation) you can provision Spacelift stacks (our industry loves this word, huh?) for each component in your stack.
+
+## Catalogs
+
+Catalogs in SweetOps are collections of sharable and reusable configuration. These are typically YAML configurations that can be imported and provide solid baselines to configure security, monitoring, or other 3rd party tooling.
+
+Today SweetOps provides a couple important catalogs:
+
+1. [DataDog Monitors](https://github.com/cloudposse/terraform-datadog-monitor/tree/master/catalog/monitors): Quickly bootstrap your SRE efforts by utilizing some of these best practice DataDog application monitors.
+1. [AWS Config Rules](https://github.com/cloudposse/terraform-aws-config/tree/master/catalog): Quickly bootstrap your AWS compliance efforts by utilizing hundreds of [AWS Config](https://aws.amazon.com/config/) rules that automate security checks against many common services.
+
+In the future you're likely to see additional open source catalogs for OPA rules or some other custom tool that requires configuration that can be easily shared. But it is important to note that catalogs are really up to you to define your own that are specific to your organization.
+
+## Primary vs Delegated
+
+Primary vs Delegated is a common implementation pattern in SweetOps. This is most easily described when looking at the example of domain and DNS usage in a mutli-account AWS organization: SweetOps takes the approach that the root domain (e.g. `example.com`) is owned by a **primary** AWS account where the root hosted zone resides. Subdomains on that domain (e.g. `dev.example.com`) are then provided to other **delegated** AWS accounts via creating an NS record on the primary hosted zone which points to the delegated hosted zone.
+
+You can see examples of this pattern in the [dns-primary](https://github.com/cloudposse/terraform-aws-components/tree/master/modules/dns-primary) / [dns-delegated](https://github.com/cloudposse/terraform-aws-components/tree/master/modules/dns-delegated) and [iam-primary-roles](https://github.com/cloudposse/terraform-aws-components/tree/master/modules/iam-primary-roles) / [iam-delegated-roles](https://github.com/cloudposse/terraform-aws-components/tree/master/modules/iam-delegated-roles) components.
+
+## Docker Based Infrastructure Development
+
+In the landscape of developing infrastructure, there are a dozen+ tools that we all need on our personal machines to do our jobs. In SweetOps, instead of having you install each tool individually we use Docker to package all of these tools into one image that you can use as your infrastructure automation toolbox: [Geodesic]({{< relref "reference/tools.md#geodesic" >}}).
```

Comment:
```suggestion
In the landscape of developing infrastructure, there are dozens of tools that we all need on our personal machines to do our jobs. In SweetOps, instead of having you install each tool individually, we use Docker to package all of these tools into one image that you can use as your infrastructure automation toolbox: [Geodesic]({{< relref "reference/tools.md#geodesic" >}}). We use `geodesic` as the base Docker image of our `Dockerfile` for all our DevOps tooling.
```